marklogic-data-hub-central/ui/src/components/advanced-settings/advanced-settings.tsx

```diff
@@ -870,11 +908,15 @@ const AdvancedSettings: React.FC<Props> = (props) => {
                 aria-label="headers-textarea"
                 style={!headersValid ? {border: "solid 1px #C00"} : {}}
               />
-              {!headersValid ? <div className={styles.invalid}>{invalidJSONMessage}</div> : null}
-              <div className={styles.textareaTooltip}>
-                <Tooltip title={tooltips.headers} placement={"right"}>
-                  <Icon type="question-circle" className={styles.questionCircle} theme="filled" />
-                </Tooltip>
+              { !headersValid ? <div className={styles.invalid}>{invalidJSONMessage}</div> : null }
+              <div className={styles.textareaHCTooltip}>
+                <HCTooltip
+                  text={tooltips.headers}
+                  id="headers-tooltip"
+                  placement="left"
```

Comment:
@federicoRufrancosMarklogic  I see in different files the value for `placement` is the opposite of what it used to be. 
Can you please double check all the `placement` values for all the ToolTips?

Reply:
@romanlescano double check it, and It's actually the opposite from an Ant library:
Ant placement="right" is equivalent to Bootstrap placement="left".
If somewhere is the same, that's a warning, and will be re-review it

Reply:
The thing is on Ant, when the tooltip doesn't tip on the placement defined, it's placed on the opposite side. So as the advanced settings is been displayed on the modal, And decided to show it on the left side
![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/88794866/131897368-33bcd4b8-844a-4ea9-8299-c963bb8d28c7.png)

And if I change the input width, its located on the placement specified
![image](https://user-images.githubusercontent.com/88794866/131897159-9062c0b5-6d24-47f5-94e1-95a288988994.png)